### DISCUSSION TITLE
Pull Request Tab in Team Page

### DISCUSSION THREAD

**Discussion by destumme** [community/community#109719] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Adding a tab to the Team pages that show a list of all PRs with the team requested as a reviewer would be incredibly useful for managing pull requests within a team directly in github. 

Similar to the notifications view for a single user, a table view with what user(s) have given a review on the PR, the most recent action run status, etc, would help in ensuring prs are not missed and that individual team members are not picking up a pr and starting a review that another team member is already taking a look at.

This can be achieved to a certain degree with projects and auto adding issues from individual repos, but for teams that work across many other teams, individually adding repos constantly to the project workflow boards would be tedious. There is also the notification filter for "Team mentioned", but review requests don't seem to be "mentions" as far as I can tell. And the "Review Requested" filter you cannot tell if that was a personal review request or a team request from what I can see.

**Comment by ettaboyle** [comment]
@destumme  Thank you for sharing this! I've added the `enhancement` label so this will be shared with the appropriate team as a feature request / enhancement.

In case you're not aware / FYI:

> This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.
> 
> If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."
> 

Thank you again for contributing and continuing to support other users in the GitHub Community Discussions.
